### PR TITLE
e2e: move rawexec oversub tests into oversubscription e2e test suite

### PR DESCRIPTION
**Pull request by shoenig**
This PR moves two tests for raw_exec and memory oversubscription into
the oversubscription test suite, which has the necessary plumbing to
activate and restore the oversubscription configuration of the scheduler
during the test.

### CODE REVIEW

File: e2e/oversubscription/input/oversub.hcl
Comment (gulducat) [inline e2e/oversubscription/input/oversub.hcl:1]: `oversub.hcl` is vague in this new context -- others in this dir include `docker.hcl` and `exec.hcl` so maybe these two could be `raw_exec.hcl` and `raw_exec_max.hcl`?

**Review by gulducat — APPROVED** [review]
more specific filenames would be nice, but LGTM!

File: e2e/oversubscription/input/oversub.hcl
Comment (shoenig) [reply e2e/oversubscription/input/oversub.hcl:1]: You and your reasonable demands!

Comment (gulducat) [reply e2e/oversubscription/input/oversub.hcl:1]: I'm glad you correctly interpreted my "maybe" and "could be" as "I insist" and "_must_ be" (and a heavily implied "_or else._")

**Pull request merged**